drivers/led/pca_common.c

```diff
@@ -60,6 +66,12 @@ LOG_MODULE_REGISTER(pca_common);
 #define PCA963X_MIN_BRIGHTNESS  0U
 #define PCA963X_MAX_BRIGHTNESS  100U
 
+/* PCA9956 parameters */
+#define PCA9956_MIN_PERIOD	67U
+#define PCA9956_MAX_PERIOD	16776U
+#define PCA9956_MIN_BRIGHTNESS	0U
+#define PCA9956_MAX_BRIGHTNESS	100U
```

Comment:
It seems like you could save a little more ROM by initializing one `struct led_data` per family, and pointing to it from each instance:

```
struct pca_common_config { /* ... */ const struct led_data *led_data; /* ... */ };
```

But this is not a blocking comment, please feel free to disregard this idea or defer it to a later pull request.